core/src/main/java/com/nisovin/magicspells/util/managers/BuffManager.java

```diff
@@ -71,12 +71,13 @@ private class BuffMonitor implements Consumer<ScheduledTask>, Listener {
 		private final ScheduledTask task;
 
 		private BuffMonitor() {
-			task = Bukkit.getGlobalRegionScheduler().runAtFixedRate(MagicSpells.getInstance(), this, interval, interval);
+			if (interval > 0) task = Bukkit.getGlobalRegionScheduler().runAtFixedRate(MagicSpells.getInstance(), this, interval, interval);
```

Comment:
[This is already checked. The constructor isn't called if `interval > 0`.](https://github.com/TheComputerGeek2/MagicSpells/pull/986/files#diff-2cceb8094500aaa6e155da613c0efba2db865be205c884f55261080b954636e0R41)